test-case/check-sof-logger.sh

```diff
@@ -71,7 +71,14 @@ sof_alsa_card_found()
     # note: assumes SOF card names to start with "sof", e.g.
     #   - /proc/asound/sofsoundwire/id
     #   - /proc/asound/sofhdadsp/id
-    test -e /proc/asound/sof*/id
+    # test -e /proc/asound/sof*/id
+    # - https://github.com/thesofproject/sof-test/issues/1243
+    # Internal issue #615
```

Comment:
I'd avoid these internal references. At least should mention "Intel internal issue tracker".

Reply:
I don't think any reference or comment is required _in the source_ in this particular case. It's pretty obvious a `for` loop is designed to support multiple devices.

If you really want to add a comment, say something like `# we have multiple sof* devices every time we enable sofprobes`. When possible, it's better not to have to open a link to understand something. References in the source are for complicated problems that are too long to explain in place. Otherwise the commit message is enough: you can have as many references as you want in the git log.